src/server/api/routes/router.js

```diff
@@ -34,4 +35,8 @@ router.use("/vessels", vesselRouter);
 
 // /api/voyages
 router.use("/voyages", voyagesRouter);
+
+// /api/favorite_vessels/
+router.use("/favorite_vessels", favoriteVesselsRouter);
```

Comment:
Update route to `/api/favorite-vessels`.

Sorry, that's my bad. I was not consistent when I specified this. But I would like to use dashes to separate words in filenames, dashes in urls, camelCase in javascript and underscores in the DB. I'm going to make a separate post about that in the channel.

Reply:
Yes sir ;)